controllers/util/mw_util.go

```diff
@@ -284,9 +284,7 @@ func (mwu *MWUtil) CreateOrUpdateNamespaceManifest(
 	manifestWork := mwu.newManifestWork(
 		mwName,
 		managedClusterNamespace,
-		map[string]string{
-			OCMBackupLabelKey: OCMBackupLabelValue,
```

Comment:
Is this commit required in this PR? Can we have a different one.

Reply:
This is corrected @raghavendra-talur if you ack we can merge, Thanks!